legacy/ui/legacy/src/main/java/com/fsck/k9/ui/settings/general/GeneralSettingsFragment.kt

```diff
@@ -37,6 +43,21 @@ class GeneralSettingsFragment : PreferenceFragmentCompat() {
         setHasOptionsMenu(true)
         setPreferencesFromResource(R.xml.general_settings, rootKey)
 
+        val disableFontSizeConfig = when (featureFlagProvider.provide("disable_font_size_config".toFeatureFlagKey())) {
+            FeatureFlagResult.Enabled -> true
+            FeatureFlagResult.Disabled -> false
+            FeatureFlagResult.Unavailable -> false
+        }
+
+        if (disableFontSizeConfig) {
+            val parentPreference = findPreference<PreferenceCategory>("global_preferences")
+            val fontSizePreferenceScreen = findPreference<PreferenceScreen>("font_size")
+
+            if (parentPreference != null && fontSizePreferenceScreen != null) {
+                parentPreference.removePreference(fontSizePreferenceScreen)
+            }
+        }
```

Comment:
Could be replaced by:

```suggestion
        featureFlagProvider.provide("disable_font_size_config".toFeatureFlagKey())
            .onEnabled {
                val parentPreference = findPreference<PreferenceCategory>("global_preferences")
                val fontSizePreferenceScreen = findPreference<PreferenceScreen>("font_size")

                if (parentPreference != null && fontSizePreferenceScreen != null) {
                    parentPreference.removePreference(fontSizePreferenceScreen)
                }
            }
```